plugins/org.eclipse.emf.mwe.core/pom.xml

```diff
@@ -8,7 +8,7 @@
 	</parent>
 
 	<artifactId>org.eclipse.emf.mwe.core</artifactId>
-	<version>1.5.0-SNAPSHOT</version>
+	<version>1.5.0</version>
```

Comment:
change intended?

Reply:
Yes and no. I added this to provoke the warning message. But this should not be merged. I'll revert that now.